src/components/Section/Section.css

```diff
@@ -63,10 +67,29 @@
 }
 
 @media screen and (max-width: 768px) {
-
   .section_layout_vertical {
     flex-direction: column;
   }
+
+  .section_type_products {
+    height: 986px;
+    position: relative;
+  }
+
+  .section__title_type_products {
+    text-align: center;
+    margin: 514px auto 0;
+  }
+}
+
+@media screen and (max-width: 600px) {
+  .section__title {
+    margin: auto;
+    margin-bottom: 82px;
+    font-size: 28px;
+    line-height: 32px;
+    text-align: center;
+  }
 }
```

Comment:
Layout breaks between 320 and 768

![image](https://user-images.githubusercontent.com/37159177/112776519-e53eba00-900d-11eb-894a-f3366b2fcb71.png)